sdk/tests/unit/TestBaseUsingEnvVars.cpp

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (c) 2023 Robert Bosch GmbH
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "TestBaseUsingEnvVars.h"
+
+#include <cstdlib>
+#include <string>
+
+namespace velocitas {
+
+void TestUsingEnvVars::checkToStoreVariableState(const std::string& varName) {
```

Comment:
A rather complex method name. Can we think of something better?

E.g.

```suggestion
void TestUsingEnvVars::preserveEnvVarState(const std::string& varName) {
```